resolver/hosts_file_resolver.go

```diff
@@ -0,0 +1,182 @@
+package resolver
+
+import (
+	"fmt"
+	"net"
+	"os"
+	"strings"
+
+	"github.com/0xERR0R/blocky/model"
+	"github.com/0xERR0R/blocky/util"
+	"github.com/miekg/dns"
+	"github.com/sirupsen/logrus"
+)
+
+const hostsDNSTTL = 3600
```

Comment:
Please externalize this parameter like "CustomTTL" with defined default value. See https://github.com/0xERR0R/blocky/blob/9259c5cdcdccbda7fd243d33aa564f0153d8c0ef/config/config.go#L313

Reply:
Done